lib/ansible/modules/wait_for_connection.py

```diff
@@ -108,6 +108,11 @@
     ansible.builtin.wait_for_connection:
       timeout: 900
 
+  - name: Wait for server to become reachable, polling every 10 seconds
+    ansible.builtin.wait_for_connection:
+      timeout: 300
+      sleep: 10
+
```

Comment:
```suggestion
  - name: Wait for server to become reachable, polling every 10 seconds
    ansible.builtin.wait_for_connection:
      timeout: 300
      sleep: 10

```

Move this example to line 75. This example does not make any sense here.